crates/katana/primitives/src/genesis/json.rs

```diff
@@ -99,6 +99,15 @@ pub struct GenesisClassJson {
     /// The class hash of the contract. If not provided, the class hash is computed from the
     /// class at `path`.
     pub class_hash: Option<ClassHash>,
+    #[serde(default, skip_serializing)] 
+    name: Option<String>,
```

Comment:
Why did you choose to ignore this field during serialization? The name should remain if we write a genesis file I suppose.